GoogleForCreators/web-stories-wp
# Task: Clean-up Dialog Props

## Context

Last sprint when the redesign was done there was a little bit of hold over in prop naming for dialogs in the editor, this is cleaning that up for uniformity going forward. 

Wasn't in scope but I had a little time this afternoon and am blocked on feature work, @samwhale pointed out this inconsistency in a new dialog for the prepublish checklist, decided to fix it rather than kick it further down the road.

## Summary

Renaming `open` to `isOpen`, restoring global modal styles, and removing redundant modal component.

## Relevant Technical Choices

Nothing profound.

## To-do



## User-facing changes

Should see no changes.

## Testing Instructions

This will involve testing all of the dialogs in the editor, fun! 🙃  

1. Grid view for carousel 
2. Post Publish 
3. Preview Error 
4. Title Missing on Publish 
5.  Delete media 
6. Update media 
7. Media permissions (clear cache then go to new story -> 3p media tab to see this one)
8. Video optimization (make sure the experiment is on, clear cache, upload a video to the editor by dragging it in) 
9. Post lock 
10. Post takeover 
11. Status Check Failed 

(you can also look at them on storybook)

### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #
